fiber/sentryfiber.go

```diff
@@ -29,28 +34,26 @@ type handler struct {
 
 type Options struct {
 	// Repanic configures whether Sentry should repanic after recovery, in most cases it should be set to false,
-	// as fasthttp doesn't include it's own Recovery handler.
+	// as fasthttp doesn't include its own Recovery handler.
 	Repanic bool
 	// WaitForDelivery configures whether you want to block the request before moving forward with the response.
-	// Because fasthttp doesn't include it's own Recovery handler, it will restart the application,
+	// Because fasthttp doesn't include its own Recovery handler, it will restart the application,
 	// and event won't be delivered otherwise.
 	WaitForDelivery bool
 	// Timeout for the event delivery requests.
 	Timeout time.Duration
 }
 
 func New(options Options) fiber.Handler {
-	handler := handler{
-		repanic:         options.Repanic,
-		timeout:         time.Second * 2,
-		waitForDelivery: options.WaitForDelivery,
+	if options.Timeout == 0 {
+		options.Timeout = 2 * time.Second
 	}
 
-	if options.Timeout != 0 {
-		handler.timeout = options.Timeout
-	}
-
-	return handler.handle
+	return (&handler{
+		repanic:         options.Repanic,
+		timeout:         options.Timeout,
+		waitForDelivery: options.WaitForDelivery,
+	}).handle
```

Comment:
This reduces readlbility

Reply:
It is the same in Echo, Gin and iris. Just made it consistent.